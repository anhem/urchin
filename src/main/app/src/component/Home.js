import React, {Component} from 'react'

class Home extends Component {

    render() {
        return (
            <div data-view="home"
                 className="home">
                Home
            </div>
        )
    }
}

export default Home