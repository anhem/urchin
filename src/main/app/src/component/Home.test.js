import React from 'react';
import {mount} from 'enzyme'
import Home from "./Home";

describe('Home', () => {

    it('renders without crashing', () => {
        expect(mount(<Home/>).length).toEqual(1);
    });
});
