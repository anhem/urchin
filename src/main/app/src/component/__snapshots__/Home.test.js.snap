// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home match snapshot 1`] = `
<div
  id="home"
>
  Home
</div>
`;
